config/env.config.js

```diff
@@ -55,5 +57,77 @@ const config = {
     }
 };
 
+const RedirectPlugin = function(options) { 
```

Comment:
Umm I should have worded it better. Declare RedirectPlugin as a class instead and use the method notation for `apply`. 
Inside of `apply` for the compiler.plugin use lambdas throughout.